internal/clientconn/conn.go

```diff
@@ -531,9 +534,11 @@ func (c *conn) route(connCtx context.Context, reqHeader *wire.MsgHeader, reqBody
 			closeConn = true
 			result = "unhandled"
 
-			c.l.Desugar().Error(
+			c.l.ErrorContext(
+				connCtx,
 				"Handler error for unhandled response opcode",
-				zap.Error(err), zap.Stringer("opcode", resHeader.OpCode),
+				logging.Error(err),
+				slog.Any("opcode", resHeader.OpCode),
```

Comment:
That should be
```go
slog.String("opcode", resHeader.OpCode.String())
```

Please fix another place too

Reply:
Thanks, this and other places addressed in https://github.com/FerretDB/FerretDB/pull/4471